From 8c3e6633e3c901bc5a5cd477383421b0980dcd26 Mon Sep 17 00:00:00 2001
From: Alexey Sokolov <sokolov@google.com>
Date: Mon, 17 Apr 2023 23:43:59 +0100
Subject: [PATCH] Fix #278

There's no real reason to install the same headers twice, but at least
now clients of the library can build again
---
 QXlsx/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/QXlsx/CMakeLists.txt b/QXlsx/CMakeLists.txt
index 4fa7f7f..074922a 100644
--- a/QXlsx/CMakeLists.txt
+++ b/QXlsx/CMakeLists.txt
@@ -179,7 +179,7 @@ PRIVATE
     ${QXLSX_HEADERPATH}
 PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/header>
-    $<INSTALL_INTERFACE:include/QXlsx>
+    $<INSTALL_INTERFACE:include/QXlsxQt${QT_VERSION_MAJOR}>
 )
 
 set_target_properties(QXlsx PROPERTIES
